examples/ale/train_nsq_ale.py

```diff
@@ -87,6 +88,9 @@ def make_env(process_idx, test):
             episode_life=not test,
             clip_rewards=not test)
         env.seed(int(env_seed))
+        if test:
+            # Randomize actions like epsilon-greedy in evaluation as well
+            env = chainerrl.wrappers.RandomizeAction(env, 0.05)
```

Comment:
I did so just because the previous `examples/ale/train_nsq_ale.py` did so. I agree that it's better to allow configuring it, but since this is not relevant to this PR, I kept it unchanged.